msat/src/msat_solver.cpp

```diff
@@ -252,6 +264,28 @@ Term MsatSolver::get_value(Term & t) const
   return Term(new MsatTerm(env, val));
 }
 
+TermVec MsatSolver::get_unsat_core()
+{
+  TermVec core;
+  size_t core_size;
+  msat_term * mcore = msat_get_unsat_assumptions(env, &core_size);
+  if (!mcore || !core_size)
+  {
+    throw InternalSolverException(
+        "Got an empty unsat core. Ensure your last call was unsat and had "
+        "assumptions in check_sat_assuming that are required to get an unsat "
+        "result");
+  }
+  msat_term * mcore_iter = mcore;
+  for (size_t i = 0; i < core_size; ++i)
+  {
+    core.push_back(std::make_shared<MsatTerm>(env, *mcore_iter));
+    mcore_iter++;
```

Comment:
minor: change it to `++mcore_iter;`

Reply:
Done 311fd024b17bc15188dc5908e971282911afa01a